src/features/patients/pages/PatientForm.tsx

```diff
@@ -14,6 +15,19 @@ import { usePatientSchemaQuery } from "../hooks/usePatientSchemaQuery";
 import { usePatientApi } from "../hooks/usePatientApi";
 import { usePatientEvent } from "../hooks/usePatientEvent";
 
+const ajvErrors = require("ajv-errors");
+
+const ajv = new Ajv({
+  errorDataPath: "property",
+  allErrors: true,
+  multipleOfPrecision: 8,
+  schemaId: "auto",
+  unknownFormats: "ignore",
+  jsonPointers: true,
+});
```

Comment:
Need to tidy up all this validator usage

Reply:
could this little guy be wrapped up in a hook? The syntax of `useAjvErrors` sounds neat.